assets/sass/base/_defaults.scss

```
@@ -141,9 +141,8 @@
 		display: none;
 	}
 
-	legend,
-	p {
-		font-size: 1rem;
+	p:not([class*="mdc-"]) {
+		font-size: 0.75rem;
```

@hussain-t The font size should be left unchanged for all `p` tags that do not start with "mdc-" :)
```suggestion
		font-size: 1rem;
```

That means for all p tags having `mdc-*` class names, they will inherit the font size from the default MDC styles.

---

Thanks @asvinb I have addressed the changes and updated the VRT files.